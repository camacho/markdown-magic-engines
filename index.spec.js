const format = require('./index.js');

describe('markdown-magic-engines', () => {
  it('renders engines from nearest package.json', () => {
    expect(format('foo', {}, { originalPath: __dirname })).toMatchSnapshot();
  });
});
