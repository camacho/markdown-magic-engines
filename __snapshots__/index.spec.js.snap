// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`markdown-magic-engines renders engines from nearest package.json 1`] = `"* **node**: ^8.0.0"`;
